addons/sale_loyalty/tests/test_program_numbers.py

```diff
@@ -687,6 +687,7 @@ def test_coupon_and_coupon_discount_fixed_amount_tax_excl(self):
                 'discount': 288.5,
                 'discount_applicability': 'order',
                 'required_points': 1,
+                'tax_ids': [(6, 0, (self.tax_15pc_excl.id,))],
```

Comment:
You shouldn't change a test setup unless really necessary, or it won't test the same things.
In this case, you should add specific tests for taxes management on 'per_point' and 'per_order' rewards, but only adapt the expected values in the existing tests IMO.